src/views/CompareWithAzion/CompareView.vue

```diff
@@ -0,0 +1,250 @@
+              </div>
+            </div>
+          </div>
+
+          <div>
+            <Message
+              v-if="errorDataWithAzion"
+              severity="error"
+              :closable="false"
+            >
+              {{ errorDataWithAzion }}
+            </Message>
+
+            <div v-if="!errorDataWithAzion">
+              <h2 class="text-3xl">Performance <span style="color: #f3652b">with Azion</span></h2>
+
+              <div class="mt-12">
+                <h3 class="text-xl mb-8">General</h3>
+                <ResumeBlock :data="resumeDataWithAzion" />
+              </div>
+
+              <div class="mt-12">
+                <h3 class="text-xl mb-8">Page Performance</h3>
+                <GradeBlock :items="gradeDataWithAzion" />
+              </div>
+
+              <div class="mt-12">
+                <h3 class="text-xl mb-8">Security</h3>
+                <SecurityBlock :securityHeaders="secureDataWithAzion" />
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <JourneyBlock class="mt-12" />
+      </template>
+    </ContentBlock>
+  </div>
+</template>
+
+<script setup>
+  import { onBeforeMount, ref } from 'vue'
+  import Message from 'primevue/message'
+
+  import PageHeadingBlock from '@/templates/page-heading-block'
+  import ContentBlock from '@/templates/content-block'
+
+  import GradeBlock from './blocks/grade-block.vue'
+  import SecurityBlock from './blocks/security-block.vue'
+  import JourneyBlock from './blocks/journey-block.vue'
+  import ResumeBlock from './blocks/resume-block.vue'
+
+  import { extract } from './utils/result-extract'
+  import { convertMsToSeconds } from './utils/convert-ms-sec'
+  import { formatBytesToKB } from './utils/format-bytes-to-kb'
+  import { resultparse } from './utils/result-data-parse'
+
+  import { useAccountStore } from '@/stores/account'
+  const accountStore = useAccountStore()
+  const { client_id } = accountStore.account
+
+  const props = defineProps({
+    testConsolidationService: {
+      type: Function,
+      required: true
+    },
+    getTestById: {
+      type: Function,
+      required: true
+    },
+    getAllTestsByClientId: {
+      type: Function,
+      required: true
+    },
+    getResult: {
+      type: Function,
+      required: true
+    },
+    getResultFromWebpagetest: {
+      type: Function,
+      required: true
+    }
+  })
+
+  const generalError = ref('')
+  // Origin Test
+  const errorData = ref('')
+  const resumeData = ref({})
+  const gradeData = ref([])
+  const secureData = ref({})
+  // Origin Test With Azion
+  const errorDataWithAzion = ref('')
+  const resumeDataWithAzion = ref({})
+  const gradeDataWithAzion = ref([])
+  const secureDataWithAzion = ref({})
+
+  const getResultFromWebpagetest = async (id) => await props.getResultFromWebpagetest(id)
+  const getQueryString = (name) => new URLSearchParams(window.location.search).get(name)
```

Comment:
Com os ajuste no serviço este código acaba não tendo necessidade.
```suggestion
  const getResultFromWebpagetest = async (id) => await props.getResultFromWebpagetest(id)
  const getQueryString = (name) => new URLSearchParams(window.location.search).get(name)
```